tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -223,6 +223,18 @@ def test_brand_string(test_microvm_with_api, network_config):
     "0xc0010007",  # MSR_K7_PERFCTR3
     "0xc001020b",  # Performance Event Counter MSR_F15H_PERF_CTR5
     "0xc0011029",  # MSR_F10H_DECFG also referred to as MSR_AMD64_DE_CFG
+    # Some MSRs intermittently fail the wrmsr restore test.
+    # These MSRs are not updated by the test and as per definition
+    # from https://github.com/gz/rust-x86/blob/master/src/msr.rs
+    # they seem to change because they have different interrupt
+    # vector value. Since this is something we can't control right now
+    # adding these MSRs to the exception list.
+    # refer https://github.com/gz/rust-x86/blob/master/src/msr.rs
+    # for below MSR names
+    "0x830"     ,  # IA32_X2APIC_ICR, intermittent issue on T2A
```

Comment:
nit: instead of saying it's an intermittent issue, let's explain why they are volatile and should be excluded from the test.

Reply:
We have a common explanation at the beginning "they seem to change because they have different interrupt vector value" and on individual lines we have the names and the scenario where the issue is seen.

Reply:
Yeah, my point is "intermittent issue" is not very meaningful for the context the reader of this file my have. We could say something more neutral like "this MSR is dynamic/volatile on AMD Milan, hence it's excluded". In other words, there is no issue per se, it's just a property of the MSR that we need proper handling for.

Reply:
oh ok got it